_posts/2024-04-09-Financial-Algorithms.md

```diff
@@ -0,0 +1,115 @@
+
+Of course, real-world problems go beyond simple multiplication. Option pricing is another potential application for quantum computing. An option is a contract that allows but does not obligate its owner to perform some specified transaction in the future. Setting a price for an option can be difficult, as the price depends on the future value of its underlying asset and the agreed conditions of the option itself. For certain type of options, the valuation process is computationally expensive, due to the uncertainty of future markets. We chose to implement a quantum algorithm for pricing the European call option, a simple but realistic type of an option. Despite its simplicity, the implementation can be extended for more complicated options with relative ease [2]. The idea is to use MLQAE to calculate the expected value of the option's payoff function. In other words, we use a quantum computer's parallel properties to simultaneously evaluate the option's different future prices, something which classically would be done separately.
+
+Here, a reality check is in order. Both housing price estimation and option pricing encounter challenges on a real QPU. The circuits required for more complex problems are too long for current noise levels. Displayed in Figure 4 are the results of the European option pricing done with MLQAE. A simulation converges faster than the classical sampling method. However, results with a real QPU fall apart due to the qubits' loss of coherence. The good news is that these algorithms work as a concept even with quantum computers like Helmi, with only a few qubits available. Increased accuracy comes in time with more advanced hardware.
+
+<figure>
+    <img src="/assets/images/Financial-Algorithms-Blog/Figure4-Statistical-mean-of-amplitude-error.png" alt="Figure 4: Statistical-mean-of-amplitude-error">
+    <figcaption>
+    <p>
+    <em>Figure 4: Statistical mean of amplitude error for evaluating the European call option. Comparison between MLQAE algorithm on a simulator and the Helmi quantum computer, and a classical sampling method</em>
+    </p>
+    </figcaption>
+</figure>
+
+## Finance and quantum computers gather public interest
+Experts from different fields interested in gaining a quantum advantage in the future are now working to improve their knowledge and encouraging people to learn about quantum technologies.
+
+Last November, the Hanken School of Economics in Finland and Ultrahack organized a quantum hackathon for finance-oriented minds with the theme of sustainable finance in the quantum era. The competition saw many teams with different backgrounds tackling the challenge and coming up with innovative solutions to combine the financial sector with the capabilities of quantum computers. Taking part in supporting the event, CSC, with colleagues from VTT and IQM, provided participants access to the 20-qubit Leena quantum computer through the LUMI supercomputer.
```

Comment:
"Taking part in supporting the event, CSC with colleagues from VTT and IQM " -> "Taking part in supporting the event, CSC and VTT with colleagues from IQM "